src/main/java/sirius/tagliatelle/emitter/Emitter.java

```diff
@@ -101,4 +105,36 @@ public Position getStartOfBlock() {
      *                improved error messages).
      */
     public abstract void visitExpressions(@Nonnull Function<Position, ExpressionVisitor> visitor);
+
+    /**
+     * Emits escaped comments to the rendering context with DEBUG level
```

Comment:
use @link for Debug level and add '.' at end of sentence

Reply:
JavaDoc updated.